Library/Formula/dmd.rb

```diff
@@ -65,6 +57,27 @@ def install
     end
   end
 
+  # Fix up dmd.conf to point to the correct include paths.
+  def fix_dmd_conf
+    # Homebrew's etc.install method doesn't let you overwrite etc files,
+    # so edit it in place if it's there:
+    conf = etc/"dmd.conf"
+    if conf.exist?
+      inreplace conf, /^DFLAGS=.+$/, "DFLAGS=-I#{include}/d2 -L-L#{lib}"
+    else
+      conf.write <<-EOS.undent
+        [Environment]
+        DFLAGS=-I#{include}/d2 -L-L#{lib}
+        EOS
```

Comment:
Sorry, I'm a bit of a newbie to Homebrew formulas.  I don't see `opt_include` or `opt_lib` documented in https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md 

Is there another place I should look for docs/examples on those?  How will it mean that the paths are correct on upgrade?